app/forms/cip_change_alert_form.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class CipChangeAlertForm
+  include ActiveModel::Model
+
+  attr_accessor :view_guidance_option
+
+  validates :view_guidance_option, presence: { message: "TODO fill me in, I am not real copy" }
```

Comment:
I am not a real copy, but maybe the fact that this needs to be filled in can be stored in JIRA, and we can put a message here that looks reasonable to the user :D 

"Select whether you want to view guidance"?